apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java

```diff
@@ -3,7 +3,8 @@
 import android.app.Application;
 
 import com.apollographql.android.converter.pojo.ApolloConverterFactory;
```

Comment:
with latest PR merged this should be changed to:
com.apollographql.android.converter.ApolloConverterFactory